db/migrate/20160324124129_add_index.rb

```diff
@@ -0,0 +1,4 @@
+class AddIndex < ActiveRecord::Migration
+  def change
```

Comment:
I can revert the schema.rb effectively by deleting the migration and dumping it. or is there a schema rollback command?